packaging/machine-learning-api.spec

```diff
@@ -242,6 +242,7 @@ DESTDIR=%{buildroot} ninja -C build %{?_smp_mflags} install
 %files devel
 %{_includedir}/nnstreamer/nnstreamer.h
 %{_includedir}/nnstreamer/nnstreamer-single.h
+%{_libdir}/libcapi-nnstreamer.so.*
```

Comment:
In general, a **devel** package does include ```.so``` only rather than all of the ```.so.*```.

Reply:
I checked that the file and it was a symlinked to `.so.*`. should we make this as a standalone file?

Reply:
> I checked that the file and it was a symlinked to .so.*

Yes, that is what exactly .so file is.

Reply:
And the same line appears the one at 240.

Reply:
Understood and deleted. Thank you for giving me the opportunity to learn 👍